src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -20,7 +24,9 @@ public Integer oddSum(List<Integer> numbers) {
      * Вернуть количество вхождений объекта `element`</p>
      **/
     public Long elementCount(List<String> elements, String element) {
-        return null;
+        return elements.stream()
+                .filter(string -> string.contains(element))
```

Comment:
what is the difference between contains() and equals()?

Reply:
contains() means that the strings are not the same, checks wether its substring is the same comparing to the element

Reply:
I thought the task was to find the number of occurrences of the element as a substring, not the whole string
was I wrong?

Reply:
This task expected to return the count of the number of the whole string :)